### ISSUE TITLE
Using noxy to tunnel traffic over another proxy-server possible?

### ISSUE THREAD

**Issue by PatrickHuetter** [spinn3r/noxy#2]
I'm using selenium to crawl some websites. I also use different http-proxies for that. Some of them need authentication (credentials). This worked fine with phantomjs. Now i want to use google chrome with chromedriver but chrome doesn't support proxies with credentials (via the chrome.switches configuration).

After some research i thought i could use littleProxy or noxy to create a local proxy without credentials which tunnels the traffic to a public proxy with credentials and use the local proxy as proxy within chrome. 
That would be also possible with https://github.com/sjitech/proxy-login-automator i think but as my technology stack is mostly java based i think littleProxy or noxy would be the better proxy software for that.

So my question is: Is my intend possible with noxy like described?

PS. I'm crawling with multiple threads simultaneously so i have to start many local proxies on different ports. So it's a requirement for me that this is easy possible with the noxy proxy.

**Comment by burtonator** [comment]
Yup.. Should be possible.  One issue is that noxy doesn't actually have a way to specify the password but I *think* there's an authenticator in littleproxy.  

What probably needs to happen is noxy-forward and noxy-reverse need to have a configuration per backend server for the HTTP auth to use for it...

**Comment by burtonator** [comment]
We'll accept a patch or this if it's straight forward and just uses the litttle proxy feature set... both noxy-reverse and noxy-forward would need it.
